src/services/CertificationService.ts

```
@@ -0,0 +1,106 @@
+  static async addCertificate(token: string, certificateData: CertificateDetail) {
+    try {
+      const formData = new FormData();
+      formData.append('title', certificateData.title);
+      
+      if (certificateData.publishDate) {
+        formData.append('publishDate', certificateData.publishDate);
+      }
+      
+      formData.append('file', certificateData.file);
+
+      const res = await fetch(`${this.BASE_URL}/api/certificates`, {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)